exp/api/v1alpha3/azuremanagedcontrolplane_webhook_test.go

```diff
@@ -45,7 +45,7 @@ func TestDefaultingWebhook(t *testing.T) {
 	g.Expect(*amcp.Spec.LoadBalancerSKU).To(Equal("Standard"))
 	g.Expect(*amcp.Spec.NetworkPolicy).To(Equal("calico"))
 	g.Expect(amcp.Spec.Version).To(Equal("v1.17.5"))
-	g.Expect(*&amcp.Spec.SSHPublicKey).NotTo(BeEmpty())
+	g.Expect(&amcp.Spec.SSHPublicKey).NotTo(BeEmpty())
```

Comment:
Perhaps, this will do the trick.
```suggestion
	g.Expect(amcp.Spec.SSHPublicKey).NotTo(BeEmpty())
```